packages/zowe-explorer-api/__tests__/__unit__/profiles/ZoweExplorerZosmfApi.unit.test.ts

```diff
@@ -57,6 +57,26 @@ describe("ZosmfUssApi", () => {
         jest.clearAllMocks();
     });
 
+    describe("_getSession", () => {
+        const exampleProfile = {
+            message: "",
+            type: "zosmf",
+            failNotFound: false,
+            name: "test.zosmf",
+            profile: {
+                host: "localhost",
+                password: "password",
+                protocol: "http",
+                user: "aZosmfUser",
+            },
+        } as zowe.imperative.IProfileLoaded;
+
+        it("should include protocol in the built session object", () => {
+            const api = new ZosmfUssApi();
+            expect((api as any)._getSession(exampleProfile).mISession.protocol).toBe("http");
+        });
```

Comment:
Should we expand this test to verify that all the properties of `exampleProfile.profile` have been loaded?